custom/icds_reports/tasks.py

```diff
@@ -967,6 +973,90 @@ def collect_inactive_awws():
     celery_task_logger.info("Ended updating the Inactive AWW")
 
 
+@periodic_task(run_every=crontab(day_of_week=6, hour=18, minute=30), acks_late=True, queue='icds_aggregation_queue')
```

Comment:
It still not confirmed When do we want to run this in the week, So by then please dont merge until I get the confirmation.

Reply:
Hmm. thats a good thought. I think it should not hinder  the aggregation as this query does not work on dashboard tables. But for the same reason that makes sense to not be in 'aggregation_queue'. Will change it to backgroun_queue. @czue